hasselhoffme.go

```diff
@@ -2,13 +2,19 @@ package main
 
 import (
 	"github.com/reujab/wallpaper"
+	"github.com/ervitis/hasselhoffme/images"
```

Comment:
I think this import should be from this repository.

Reply:
This is how golang as far as I know works with its packages imports